src/scene/scene_forwardrenderer.js

```diff
@@ -631,7 +636,31 @@ pc.extend(pc, function () {
             drawCalls.sort(sortDrawCalls);
 
             // Render a depth target if the camera has one assigned
-            if (camera._depthTarget) {
+            if (camera._renderDepthRequests) {

```

Comment:
Original post effect system internally sets depthMap variable for each effect that requires depth though.